pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator_test.go

```diff
@@ -221,7 +221,7 @@ var _ = Describe("VirtualMachineInstance Mutator", func() {
 			Expect(vmiSpec.Domain.CPU.Model).To(Equal(""))
 		}
 
-		Expect(vmiSpec.Domain.Resources.Requests.Cpu().String()).To(Equal("100m"))
+		Expect(vmiSpec.Domain.Resources.Requests.Cpu().IsZero()).To(BeTrue())
```

Comment:
Why do we change this?

Reply:
So the default will be calculated and normalized according to the cpu allocation ratio.